extensions/dicom-seg/src/components/SegmentationPanel.js

```diff
@@ -0,0 +1,473 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import cornerstoneTools from 'cornerstone-tools';
+import cornerstone from 'cornerstone-core';
+import moment from 'moment';
+
+import { utils } from '@ohif/core';
+import { ScrollableArea, TableList, Icon } from '@ohif/ui';
+
+import {
+  BrushColorSelector,
+  BrushRadius,
+  SegmentationItem,
+  SegmentItem,
+  SegmentationSelect,
+} from './index';
+
+import './SegmentationPanel.css';
+import SegmentationSettings from './SegmentationSettings';
+
+const { studyMetadataManager } = utils;
+
+/* TODO: wrap get/set interactions with the cornerstoneTools
+ * store with context to make these kind of things less blurry. */
+const segmentationModule = cornerstoneTools.getModule('segmentation');
```

Comment:
I think this is fine. `getModule` doesn't return a copy, you actually have "the module", so values should stay fresh. Trying to think if this is better to grab as needed.

Reply:
Grabbing as needed is probably better practice. In the general case modules might be added after cornerstoneTools has initialised, just this is one of the few "core" modules, so its there from the get-go.

Reply:
Since this is the module, i think its fine to have it on top level inside the component. Updated.